ci-scripts/publish.sh

```diff
@@ -21,5 +21,8 @@ npm publish
 # run this after publish to make sure GitHub finishes updating from the push
 yarn release:create -- --repo $TRAVIS_REPO_SLUG --tag $release_tag --branch master
 
+# build the documentation site
+yarn build:docs
+
 # deploy documentation site to github pages branch
-yarn deploy:docs
+npm run deploy:docs -- --repo "https://$GH_TOKEN@github.com/$TRAVIS_REPO_SLUG"
```

Comment:
I'm perfectly fine going back to yarn here, but I wasn't sure if it was possible to pass an argument in this way. Let me know if you want this changed to be consistent.

Reply:
I was unable to determine why `deploy:docs` failed all the time. So - Thank you very much! I have no strong feelings w/r to yarn. :)